src/main/java/inventorysetups/InventorySetupsPersistentDataManager.java

```diff
@@ -231,6 +278,28 @@ private void handleMigrationOfOldData()
 //				log.info("Removing old data key");

```

Comment:
old v1 data key

Reply:
done; also added for the post-v3 migration